satellite/overlay/service.go

```diff
@@ -77,7 +77,9 @@ type DB interface {
 	GetExitingNodes(ctx context.Context) (exitingNodes storj.NodeIDList, err error)
 	// GetExitingNodesLoopIncomplete returns exiting nodes who haven't completed the metainfo loop iteration.
 	GetExitingNodesLoopIncomplete(ctx context.Context) (exitingNodes storj.NodeIDList, err error)
-
+	// GetGracefulExitNodes returns nodes who have either (initiated and not completed) or (initiated and completed) graceful exit within a time window.
+	GetGracefulExitNodes(ctx context.Context, completed bool, begin, end time.Time) (exitingNodes storj.NodeIDList, err error)
```

Comment:
I feel I would be confused with this method and `GetExitingNodes`. Maybe we should combine these two?

Reply:
Yeah, I do not really like the name. But we do need a separate method because we need the `begin` and `end` for the window of nodes to look for

Reply:
maybe you could call it ListExitingNodes

Reply:
Maybe `GetExitingNodesByTimeFrame` ?

Reply:
I will change to `GetGracefulExitNodesByTimeFrame` for now